api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -1274,6 +1274,9 @@ components:
         phone:
           type: string
           description: Phone number of reporter.
+        sharing_allowed:
+          type: boolean
+          description: States if the reporter gives permission to share this information with 3th parties
```

Comment:
Maybe rephrase:

Reporter allows their contact information to be shared (true) or not (false).

Reply:
Done